tests/containers/run_container_in_k3s.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Installs a local kubernetes engine
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use publiccloud::utils;
+use containers::k8s qw(install_k3s apply_manifest wait_for_k8s_job_complete find_pods validate_pod_log);
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $provider = get_var("KUBECTL_CLUSTER", "k3s");
+    my $image = get_var("CONTAINER_IMAGE_TO_TEST", "registry.suse.com/bci/bci-base:latest");
+
+    if ($provider eq "k3s") {
+        install_k3s();
+
+        record_info('cmd', "k3s installed");
+
+        my $cmd = '"cat", "/etc/os-release"';
+        my $job_name = "test";
+
+        my $manifest = <<EOT;
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test
+spec:
+  template:
+    spec:
+      containers:
+      - name: $job_name
+        image: $image
+        command: [ $cmd ]
+        resources:
+          limits:
+            cpu: "0.5"
+            memory: 256Mi
+          requests:
+            cpu: "0.1"
+            memory: 128Mi
+      restartPolicy: Never
+EOT
```

Comment:
This should be moved to `data/containers/`

Reply:
But this has variables, and also are using in this way on https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/cbd70fd9f5b5617d2856813ad51de446313f0226/tests/containers/run_container_in_k8s.pm#L34

Reply:
```suggestion
assert_script_run "curl " . data_url('containers/helm_test.yaml') . " -o /tmp/helm_test.yaml";
file_content_replace("/tmp/helm_test.yaml", JOB_NAME => $job_name, IMAGE => $image, CMD => $cmd);
```

Reply:
Oh thanks a lot for the help

Reply:
Confirmed with new verification run